AlphaWallet/Transfer/Views/TransitionButton.swift

```diff
@@ -18,11 +18,13 @@ public enum StopAnimationStyle {
     case shake
 }
 
+fileprivate typealias CFG = Configuration
+
 /// UIButton sublass for loading and transition animation. Useful for network based application or where you need to animate an action button while doing background tasks.
 @IBDesignable open class TransitionButton: UIButton, CAAnimationDelegate {
     var shrinkBorderColor: UIColor = .lightGray
     var shrinkBorderWidth: CGFloat = 3.0
-    var shrinkBackgroundColor: UIColor = Colors.appWhite
+    var shrinkBackgroundColor: UIColor = CFG.Color.Semantic.inverseDefaultButtonBackground
```

Comment:
This should be `.defaultButtonBackground` right?

Reply:
This is `.defaultButtonBackground`.
https://user-images.githubusercontent.com/1050309/218632800-d19ca3df-4839-4940-93f6-b0140ef276d0.mp4